### ISSUE TITLE
UnicodeEncodeError

### ISSUE THREAD

**Issue by sky-48** [SNH48Live/KVM48#5]
Traceback (most recent call last):
  File "/usr/local/bin/kvm48", line 11, in <module>
    sys.exit(main())
  File "/usr/local/lib/python3.5/dist-packages/kvm48/kvm48.py", line 381, in main
    if not os.path.exists(fullpath) or os.path.exists(
  File "/usr/lib/python3.5/genericpath.py", line 19, in exists
    os.stat(path)
UnicodeEncodeError: 'ascii' codec can't encode characters in position 25-30: ordinal not in range(128)

More info: **https://gist.github.com/sky-48/5afb5c4a98767a54ffa755c415790a60**

**Comment by zmwangx** [comment]
Your locale probably isn't what you think it is. Also, the Python in use is clearly Python 3.5. What's the output of the following command?

```sh
python3 -c 'import locale, os, sys; print("executable:", sys.executable); print("version:", sys.version_info); print("default locale:", locale.getdefaultlocale()); print("current locale:", locale.getlocale()); print("locale preferred encoding:", locale.getpreferredencoding()); print("filesystem encoding:", sys.getfilesystemencoding()); print("env:"); print("\n".join("%s: %s" % (k, v) for k, v in sorted(os.environ.items())));'
```

The output should be look somewhat like this:

```
executable: /Users/zmwang/.pyenv/versions/3.7.1/bin/python3
version: sys.version_info(major=3, minor=7, micro=1, releaselevel='final', serial=0)
default locale: ('en_US', 'UTF-8')
current locale: ('en_US', 'UTF-8')
locale preferred encoding: UTF-8
filesystem encoding: utf-8
env:
...
```

If you have sensitive information in environment variables, please scrub them.

**Comment by sky-48** [comment]
Thank you for your help. You were right, the python was 3.5 (apparently Debian doesn't have higher version in its repo) and the locale wasn't correct.

I have updated Python and adjusted locale. Now the output is the same as yours.

executable: /usr/local/bin/python3
version: sys.version_info(major=3, minor=7, micro=1, releaselevel='final', serial=0)
default locale: ('en_US', 'UTF-8')
current locale: ('en_US', 'UTF-8')
locale preferred encoding: UTF-8
filesystem encoding: utf-8
env:
...

**Comment by zmwangx** [comment]
Still not sure what's causing the encoding issue with your `/usr/bin/python3`, but the output from `/usr/local/bin/python3` looks good. Could you please reinstall KVM48 with `/usr/local/bin/pip3` and see if the problem goes away?

**Comment by sky-48** [comment]
Sorry for the confusion, the issue has been resolved after I upgraded to Python 3.7. I guess I was not explicit enough in the previous comment.

**Comment by zmwangx** [comment]
Oh okay, good to hear.
